docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.

```

Comment:
The sentence is fine, but I would suggest the following to make the sentence more clear: 

- The [TextField](#textField) class provides a control that allows single line text editing.

Kindly ignore if you don't  like it.